### ISSUE TITLE
Redo Parmake Diagram

### ISSUE THREAD

**Issue by bhuvy2** [illinois-cs241/illinois-cs241.github.io#112]
The Parmake dependency diagram is hard to read and the SVG doesn't have a solid white backing, we should fix this for accessibility reasons.

* Make the test on the flowchart bigger.
* Don't make it so long. There are a few ways around this, either split up the diagram into two images or combine horizontal and vertical scaling in one diagram
* Commit the SVG, so it can be edited later.
